src/imitation/scripts/train_adversarial.py

```diff
@@ -54,6 +51,7 @@ def train(_seed: int,
           expert_policy_plot=None,
           show_plots: bool = True,
 
+          ray_tune_reporter=None,
           ray_tune_interval: int = -1,
```

Comment:
Similar comment to `expert_demos` re: possibility of non-positive interval and non-None reporter.

Reply:
I decided to rely on Stable Baselines logging instead of Ray Tune reporting for now, so these docs/code have been deleted.